bootique/src/test/java/io/bootique/Bootique_CliOptionsIT.java

```diff
@@ -327,6 +327,38 @@ public void commandWithOptionWithDefaultValue() {
         assertEquals("val", cli.optionString("o"));
     }
 
+    @Test
+    public void commandRequiredValue() {
+        BQRuntime runtime = appManager.runtime(Bootique.app("-B", "bVal")
+                .module(b -> BQCoreModule.extend(b)
+                        .addCommand(CommandBWithRequiredValue.class)
+                ));
+        Cli cli = runtime.getInstance(Cli.class);
+        assertEquals("commandRequiredValue", cli.commandName());
+        assertEquals("bVal", cli.optionString("B"));
+    }
+
+    @Test
+    public void commandDefaultValue() {
+        BQRuntime runtime = appManager.runtime(Bootique.app("-A")
+                .module(b -> BQCoreModule.extend(b)
+                        .addCommand(CommandAWithDefaultValue.class)
+                ));
+        Cli cli = runtime.getInstance(Cli.class);
+        assertEquals("commandDefaultValue", cli.commandName());
+        assertEquals("commandVal", cli.optionString("A"));
+    }
+
+    @Test void missingCommandDefaultValue() {
+        BQRuntime runtime = appManager.runtime(Bootique.app()
+                .module(b -> BQCoreModule.extend(b)
+                        .addCommand(CommandAWithDefaultValue.class)
+                ));
+        Cli cli = runtime.getInstance(Cli.class);
+        assertNotEquals("command", cli.commandName());
+        assertNull(cli.optionString("option"));
+    }
+
```

Comment:
I think these 3 cover basic command with arg functionality, but I'm happy to write more tests if they seem necessary.